tests/release/pipelines/fbc_release.go

```diff
@@ -108,9 +108,9 @@ var _ = framework.ReleasePipelinesSuiteDescribe("FBC e2e-tests", Label("release-
 
 		AfterAll(func() {
 			if !CurrentSpecReport().Failed() {
-				Expect(devFw.AsKubeDeveloper.HasController.DeleteApplication(fbcApplicationName, devNamespace, false)).NotTo(HaveOccurred())
-				Expect(managedFw.AsKubeDeveloper.TektonController.DeleteEnterpriseContractPolicy(fbcEnterpriseContractPolicyName, managedNamespace, false)).NotTo(HaveOccurred())
-				Expect(managedFw.AsKubeDeveloper.ReleaseController.DeleteReleasePlanAdmission(fbcReleasePlanAdmissionName, managedNamespace, false)).NotTo(HaveOccurred())
+				Expect(managedFw.AsKubeAdmin.CommonController.DeleteNamespace(managedNamespace)).NotTo(HaveOccurred())
+				Expect(managedFw.SandboxController.DeleteUserSignup(managedFw.UserName)).To(BeTrue())
+				Expect(devFw.SandboxController.DeleteUserSignup(devFw.UserName)).To(BeTrue())
```

Comment:
hmm given the fact that this test is aimed to run on a staging cluster - should we really try to clean up the user signup there?

I thought what we might wanna do here is either
* remove the `if !CurrentSpecReport().Failed() {` condition to clean up apps and other resources regardless of the test result
* to the cleanup of previous Apps/ECP/RPA at the beginning of the test, i.e. in `BeforeAll`. wdyt?

Reply:
Yeah, I like that suggestion. I will amend it